src/storage/storage.ts

```diff
@@ -163,8 +163,11 @@ export class Storage {
         socketMessage(io, { message: entry.payload.value, channelAddress })
       })
       db.events.on('replicated', () => {
-        console.log('Message replicated')
-        loadAllMessages(io, this.getAllChannelMessages(db), channelAddress)
+        console.log('Messages replicated')
+      })
+      db.events.on('replicate.progress', (address, hash, entry, progress, have) => {
```

Comment:
We should check if this won't affect notifications. Especially on fresh start when a lot messages replicate and user can receive many notifications.

Reply:
I've tried this solution and it looks like it doesn't improve performance a lot